packages/components/src/components/Search/Search.tsx

```diff
@@ -153,6 +153,7 @@ export const Search = ({
                                     color="primary"
                                     variant="ghost"
                                     onPress={handleClearText}
+                                    disabled={disabled}
```

Comment:
Should we maybe instead just don't show this clear button if the search bar is disabled? 
Could potentially just add the disabled check next to the condition for rendering this on line 148 in that case. Thoughts?

Reply:
yes that is smoother and more highly advanced senior code, i agree wholeheartedly 😡